ghost/core/core/frontend/helpers/tpl/content-cta.hbs

```diff
@@ -2,13 +2,13 @@
 <aside class="gh-post-upgrade-cta">
     <div class="gh-post-upgrade-cta-content" style="background-color: {{accentColor}}">
         {{#has visibility="paid"}}
-            <h2>This post is for paying subscribers only</h2>
+            <h2>This {{resource}} is for paying subscribers only</h2>
```

Comment:
We already have the `{{#is}}` helper to determine context of whether something is a post or page. Is there a reason why we can't use the existing pattern here?

Reply:
[Updated](https://github.com/TryGhost/Ghost/pull/15471#discussion_r979900173) ✅